app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+
+    for person in people:
+        instance = Person(person["name"], person["age"])
+        for person_info in person:
+            if person[person_info] is not None \
+                    and (person_info == "wife" or person_info == "husband"):
```

Comment:
better to use `()` then `\`
```suggestion
            if (
                person[person_info] is not None 
                and (person_info == "wife" or person_info == "husband")
            ):
```